src/clojure/nrepl/util/completion.clj

```diff
@@ -52,8 +52,9 @@
 (defn static-members
   "Returns a list of potential static members for a given class"
   [^Class class]
-  (for [member (concat (.getMethods class) (.getDeclaredFields class)) :when (static? member)]
-    (.getName ^Member member)))
+  (distinct
+   (for [member (concat (.getMethods class) (.getDeclaredFields class)) :when (static? member)]
```

Comment:
Probably it's best to convert this to some `->>` pipeline. Seems the author of the original code was quite fond of `for`, but I'm definitely more fond of `map`, `filter` and friends. :-)

Reply:
Yeah, I very rarely use `for`, too. I can certainly rewrite the function with `->>`. Would you prefer that I do it in a separate commit?